htdocs/compta/facture/card.php

```diff
@@ -3929,8 +3929,9 @@ function setRadioForTypeOfInvoice() {
 		include_once DOL_DOCUMENT_ROOT.'/core/modules/facture/modules_facture.php';
 		$liste = ModelePDFFactures::liste_modeles($db);
 		if (getDolGlobalString('INVOICE_USE_DEFAULT_DOCUMENT')) {
+			$type = GETPOSTINT('type') ? GETPOSTINT('type') : $object->type;
```

Comment:
May be 
$type = GETPOSTISSET('type') ? GETPOSTINT('type') : $object->type;
is better. Should work also if $_POST['type'] is '0'

Reply:
Done